src/agent/rustjail/src/cgroups/fs/mod.rs

```diff
@@ -1320,7 +1320,7 @@ impl CgroupManager for Manager {
         let blkio_stats = if self.paths.get("blkio").is_some() {
             match Blkio().get_stats(self.paths.get("blkio").unwrap()) {
                 Ok(stat) => SingularPtrField::some(stat),
-                Err(e) => {
+                Err(_) => {
```

Comment:
what about using `e` in the warning? ` warn!(sl!(), "failed to get blkio stats: {:?}", e);`